changelog.d/15488.feature

```diff
@@ -0,0 +1 @@
+Add automatic purge after all users forgotten a room. Also add restore of purge/shutdown rooms after a synapse restart.
```

Comment:
Does this issue fix https://github.com/matrix-org/synapse/issues/4720?

(let's link it in the issue description)